templates/etc/quagga/Quagga.conf.j2

```diff
@@ -55,6 +56,11 @@ interface {{ qdi.int }}
  ip address {{ qdi.address }}/{{ qdi.cidr }}
  ipv6 nd suppress-ra
 {%     endif %}
+{%     if qdi.ospf is defined %}
```

Comment:
It might be clearer to call this `qdi.ospf_commands` or some such.  And similarly throughout the rest of the block of course (but I don't know how to multi-select lines in the Brave New (github) World, and maybe that's [not a thing here](https://github.com/rombert/ereviewboard/issues/92)).